test/libsolidity/syntaxTests/parsing/var_array.sol

```diff
@@ -2,4 +2,4 @@ contract Foo {
 	function f() { var[] a; }
 }
 // ----
-// ParserError 2314: (34-35): Expected identifier but got '['
+// ParserError 6933: (31-34): Expected primary expression.
```

Comment:
Does all these tests for var make sense anymore, since they are just parsing errors? I'm in favour of removing all but one of them.